Library/Formula/repo.rb

```diff
@@ -1,9 +1,8 @@
 class Repo < Formula
   desc "Repository tool for Android development"
   homepage "https://source.android.com/source/developing.html"
-  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo"
-  version "1.21"
-  sha256 "44d1ac8b948e6154a495469dad05870dc6182712c0610da8d255b5471dd5bcb9"
+  url "https://gerrit.googlesource.com/git-repo", :ref => 'v1.12.32', :using => :git
```

Comment:
You can stick `.git` on the end of the `googlesource` URL; I haven't come across a GoogleSource URL that doesn't support that yet. Likewise the syntax should be:

```ruby
url "https://gerrit.googlesource.com/git-repo",
     :tag => "v1.12.32",
     :revision => "CommitSHA1"
```